src/core/js/models/adaptModel.js

```diff
@@ -467,7 +467,10 @@ define([
                 try {
                     var model = Adapt.findById(id);
 
-                    if (!model.get("_isAvailable")) continue;
+                    this.stopListening(model, "change:_isComplete");
```

Comment:
I feel this needs some kind of explanatory comment... mainly based on the fact that I can't figure out why you're unregistering a listener then immediately re-registering it again!